dluhc-component-library/src/components/policyPage/PolicyPage.tsx

```diff
@@ -1,42 +1,55 @@
-const PolicyPage = () => {
+import { useEffect, useState } from "react";
+import { PolicyData } from "../policyForm/types";
+import { loadJson } from "src/utils";
+interface PolicyPageProps {
+  policyFilePath: string;
+}
+
+const DEFAULT_POLICY_DATA: PolicyData = {
```

Comment:
maybe put this in constants as `INITIAL_STATE` to match what we do in timetable form